docs/configuration.md

```diff
@@ -22,3 +22,5 @@ The following environment variables are available:
 | `BUILDRUN_MAX_CONCURRENT_RECONCILES` | The number of concurrent reconciles by the buildrun controller. A value of 0 or lower will use the default from the [controller-runtime controller Options](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/controller#Options). Default is 0. |
 | `BUILDSTRATEGY_MAX_CONCURRENT_RECONCILES` | The number of concurrent reconciles by the buildstrategy controller. A value of 0 or lower will use the default from the [controller-runtime controller Options](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/controller#Options). Default is 0. |
 | `CLUSTERBUILDSTRATEGY_MAX_CONCURRENT_RECONCILES` | The number of concurrent reconciles by the clusterbuildstrategy controller. A value of 0 or lower will use the default from the [controller-runtime controller Options](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/controller#Options). Default is 0. |
+| `KUBE_API_BURST` | Burst to use for the Kubernetes API client. See [Config.Burst](https://pkg.go.dev/k8s.io/client-go/rest#Config.Burst). A value of 0 or lower will use the default from client-go, which currently is 10. Default is 0. |
+| `KUBE_API_QPS` | QPS to use for the Kubernetes API client. See [Config.QPS](https://pkg.go.dev/k8s.io/client-go/rest#Config.QPS). A value of 0 or lower will use the default from client-go, which currently is 5. Default is 0. |
```

Comment:
I keep meaning to mentions this @adambkaplan and have finally remembered :-)

And if you have already captured it somewhere and I missed it, apologies.

But a potential aspect of the upcoming operator would be to capture the growing list of these config env vars 
in an operator's config object, and then have the operator translate to env's it sets on the controller's deployment.

I could see a pattern where a new config setting is "alpha'ed" via env vars here, and after some vetting, in a later iteration of the operator we "promote" the specific setting to the config object and expose it in a more "official" way, so to speak.

Or if we feel super confident, we make updates both here and with the operator when new things are introduced.

Reply:
+1000 , I would like to also do this via a configmap. I will generate an issue for this.

Reply:
yeah I could see mapping and/or transferring these env's to a configmap being good, but in and of itself, as well as being something that could be more consumable by the upcoming operator @qu1queee @adambkaplan

Reply:
I think for the upcoming operator, a configmap would be more handy, but I´m not very familiar with the internals of the operator atm. We can provide multiple configmaps, depending on the config type, plus a default one, this will provide a well structured separation of concerns.